utils/utils.jsx

```diff
@@ -1170,18 +1169,7 @@ export function isMobile() {
     return window.innerWidth <= Constants.MOBILE_SCREEN_WIDTH;
 }
 
-export function getUserById(userId) {
-    const state = store.getState();
-    return getUser(state, userId);
-}
-
-export function getUserByUsername(username) {
-    const state = store.getState();
-    return getUserByUsernameRedux(state, username);
-}
-
-export function getDirectTeammate(channelId) {
-    const state = store.getState();
+export function getDirectTeammate(channelId, state) {
```

Comment:
It would be nice if these functions also took `state` as the first parameter as well.